detox/test/e2e/12.animations.test.js

```diff
@@ -17,60 +14,106 @@ describe('React-Native Animations', () => {
 
   describe.each(['JS', 'Native'])('(driver: %s)', (driver) => {
+    beforeEach(async () => {
+      await device.reloadReactNative();
+      await element(by.text('RN Animations')).tap();
+    });
+
     it(`should find element`, async () => {
       await _startTest(driver);
       await expect(element(by.id('UniqueId_AnimationsScreen_afterAnimationText'))).toBeVisible();
     });
 
     it(`should detect loops with final number of iterations`, async () => {
-      await _startTest(driver, {loops: 4});
+      await _startTest(driver, { loops: 4 });
       await expect(element(by.id('UniqueId_AnimationsScreen_afterAnimationText'))).toBeVisible();
     });
 
-    it.skip(`should not wait for infinite animations`, async() => {
-      await _startTest(driver, {loops: -1});
+    it.skip(`should not wait for infinite animations`, async () => {
+      await _startTest(driver, { loops: -1 });
       await expect(element(by.id('UniqueId_AnimationsScreen_afterAnimationText'))).toBeVisible();
     });
 
     it(`should not wait during delays longer than 1.5s`, async () => {
-      await _startTest(driver, {delay: 1600});
+      await _startTest(driver, { delay: 1600 });
       await expect(element(by.id('UniqueId_AnimationsScreen_afterAnimationText'))).not.toExist();
     });
 
     it(`should wait during delays shorter than 1.5s`, async () => {
-      await _startTest(driver, {delay: 500});
+      await _startTest(driver, { delay: 500 });
       await expect(element(by.id('UniqueId_AnimationsScreen_afterAnimationText'))).toExist();
     });
 
     it(`should not wait for an animation to complete while the synchronization is disabled`, async () => {
-      await device.disableSynchronization(); 
-      await _startTest(driver, {duration: 5000});
+      await device.disableSynchronization();
+      await _delay(800);
+      await _startTest(driver, { duration: 5000 });
+      await _delay(800);
       await expect(element(by.id('UniqueId_AnimationsScreen_afterAnimationText'))).not.toExist();
 
       await device.enableSynchronization();
       await expect(element(by.id('UniqueId_AnimationsScreen_afterAnimationText'))).toExist();
     });
   });
-});
 
-describe(':android: Native animations', () => {
-  it('should expect a native android-animator animation to be short-circuited / fully-completed', async () => {
-    await device.reloadReactNative();
-    await element(by.text('Native Animation')).tap();
+  describe('detoxEnableSynchronization', () => {
+    it('should launch without synchronization for detoxEnableSynchronization 0', async () => {
+      await device.launchApp({
+        newInstance: true,
+        launchArgs: {
+          detoxEnableSynchronization: 0
+        }
+      });
+
+      // The delays are necessary since we're running without synchronization
+      await _delay(800);
+      await element(by.text('RN Animations')).tap();
+      await _delay(800);
+      await _startTest('JS', { duration: 5000 });
+      await _delay(800);
```

Comment:
Again, why the multiple delays? A 5 seconds animation should be enough, shouldn't it?